bench/lib/bench.c

```diff
@@ -1,8 +1,10 @@
 #include <sys/time.h>
 
 #include <assert.h>
+#include <inttypes.h>
```

Comment:
What do we use from `inttypes.h` which isn't in `stdint.h`?

Reply:
Quite right; nothing.  And earlier version of this file used `PRIu64`, but it's not needed now.

Reply:
Hmm, don't we still want `<stdint.h>`?  I mean, we get it via `bench.h` but I think we normally have such inclusions explicitly in the C file when the C file uses types defined in the header.